department-of-veterans-affairs/caseflow
# Implement hearing room list as a json file that may be shared by both front-end and back-end code.

Resolves #8122 

### Description
Set up the hearing room list as a json file so it may be shared by both the HearingRooms class and by the front-end modal. 

### Acceptance Criteria

### Testing Plan
1. Go to Edit Hearing Day and make sure it shows the list of rooms properly formatted.
2. Go to Rails console and type in the following commands:
- room = HearingRooms.find!("1")
- room.label
Verify it returns "1 (1W200A)"
